config/config_test.go

```diff
@@ -28,9 +28,11 @@ func TestJSONMarshalSecret(t *testing.T) {
 		S Secret
 	}
 	for _, tc := range []struct {
-		desc     string
-		data     tmp
-		expected string
+		desc      string
+		data      tmp
+		expected  string
+		trueValue bool
```

Comment:
"trueValue" isn't a very descriptive name. How about:

```suggestion
		marshalSecret bool
```